### PR TITLE
fix: linter and spacing for is_graph_bipartite.

### PR DESCRIPTION
**Pull request by fhlasek**
#### Description of Change
Fixing consistent spacing and linter warnings.

#### Checklist

- [X] Added description of change
- [X] Added file name matches [File name guidelines](https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/CONTRIBUTING.md#New-File-Name-guidelines)
- [X] Added tests and example, test must pass
- [X] Added documentation so that the program is self-explanatory and educational - [Doxygen guidelines](https://www.doxygen.nl/manual/docblocks.html)
- [X] Relevant documentation/comments is changed or added
- [X] PR title follows semantic [commit guidelines](https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/CONTRIBUTING.md#Commit-Guidelines)
- [X] Search previous suggestions before making a new one, as yours may be a duplicate.
- [X] I acknowledge that all my contributions will be made under the project's license.

Notes: 

<a href="https://gitpod.io/#https://github.com/TheAlgorithms/C-Plus-Plus/pull/1037"></a>

### CODE REVIEW

File: math/fibonacci.cpp [hunk 1]
```diff
@@ -17,8 +17,9 @@
 unsigned int fibonacci(unsigned int n) {
     /* If the input is 0 or 1 just return the same
        This will set the first 2 values of the sequence */
-    if (n <= 1)
+    if (n <= 1) {
```
Comment (Panquesito7) [inline math/fibonacci.cpp:20]: Shouldn't this file be modified in another PR?

Comment (fhlasek) [reply math/fibonacci.cpp:20]: Interesting, somehow it got automatically fixed by "clang-tidy fixes for a49ec9b" - see the commit history, not sure I fully understand how the clang-tidy is set up for this repo.

Comment (kvedala) [reply math/fibonacci.cpp:20]: This is resolved by clang-tidy, right?

Comment (kvedala) [reply math/fibonacci.cpp:20]: I set the clang-tidy up to use the config file `.clang-tidy`. I had initially setup it on all the files but removed it as it was un-necessary. Now, it gets called only on the modified C/CPP files.

File: graph/is_graph_bipartite.cpp [hunk 2]
```diff
@@ -37,127 +37,124 @@
...
+   * @namespace is_graph_bipartite
+   * @brief Functions for checking whether a graph is bipartite or not
+   */
+  namespace is_graph_bipartite {
+    /**
+     * @brief Class for representing graph as an adjacency list.
+     */
+    class Graph {
+      private:
+        int n;            /// size of the graph
```
Comment (kvedala) [inline graph/is_graph_bipartite.cpp:51]:
```suggestion
        int n;            ///< size of the graph
```
the comment style for class members is [slightly different](https://www.doxygen.nl/manual/docblocks.html#memberdoc)

File: graph/is_graph_bipartite.cpp [hunk 3]
```diff
@@ -37,127 +37,124 @@
...
-    				adj.resize(n);
-	    			side.resize(n,-1);
-    			}
+        static const int nax = 5e5 + 1;
 
-		    	void addEdge(int u, int v);     /// function to add edges to our graph
+      public:
+        /**
+         * @brief Constructor that initializes the graph on creation
+         */
```
Comment (kvedala) [inline graph/is_graph_bipartite.cpp:62]:
```suggestion
         * @param size ....
         */
```

File: graph/is_graph_bipartite.cpp [hunk 4]
```diff
@@ -37,127 +37,124 @@
...
+     *  the function returns true if the graph is a bipartite graph
+     *  the function returns false if the graph is not a bipartite graph
+     *
+     *  @details
+     *  Here, side refers to the two disjoint subsets of the bipartite graph.
+     *  Initially, the values of side are set to -1 which is an unassigned state. A for loop is run for every vertex of the graph.
+     *  If the current edge has no side assigned to it, then a Breadth First Search operation is performed.
+     *  If two neighbours have the same side then the graph will not be bipartite and the value of check becomes false.
+     *  If and only if each pair of neighbours have different sides, the value of check will be true and hence the graph bipartite.
+     *
```
Comment (kvedala) [inline graph/is_graph_bipartite.cpp:94]:
```suggestion
     *
     * @returns `true` ....
     * @returns `false` ....
```

File: graph/is_graph_bipartite.cpp [hunk 5]
```diff
@@ -37,127 +37,124 @@
...
-	 		public:
-	    		/**
-    			 * @brief Constructor that initializes the graph on creation
-    			 */
-	    		explicit Graph(int size = nax){
-    				n = size;
-    				adj.resize(n);
-	    			side.resize(n,-1);
-    			}
+        static const int nax = 5e5 + 1;
```
Comment (kvedala) [inline graph/is_graph_bipartite.cpp:57]:
```suggestion
        static const int max = 5e5 + 1;  ///< ....
```
can also use `INT_MAX`

**Comment by kvedala** [comment]
> I think the code formatting got affected, maybe because the GitHub Actions is still not running on the fork.
> Others are just doc corrections.
> Looks good otherwise. I have not executed the code.

GitHub Actions is running, but the code formatting is not getting fixed.  Will look into it

**Comment by kvedala** [comment]
> > I think the code formatting got affected, maybe because the GitHub Actions is still not running on the fork.
> > Others are just doc corrections.
> > Looks good otherwise. I have not executed the code.
> 
> GitHub Actions is running, but the `clang-tidy` did not fix the code formatting. Will look into it

@fhlasek the fix has been pushed. can you merge there master branch to your fork
